SPECS/python-tensorboard/genereate_source_tarball.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# output = tensorboard-$TB_VERSION.tar.gz
+# Tensorboard version to use
+TB_VERSION="2.16.2"
+
+docker build --build-arg TB_VERSION=$TB_VERSION -t tensorboard .
+
+CONTAINER_ID=$(docker run -d tensorboard)
+
+docker cp $CONTAINER_ID:/root/tensorboard-$TB_VERSION.tar.gz $PWD
+
+docker stop $CONTAINER_ID
+docker rm $CONTAINER_ID
```

Comment:
I don't think this format will work with our upgrade automation. Please sync with Nicolas but I think at the minimum we need these scripts to accept the `srcTarball`, `outFolder`, and `pkgVersion` arguments. See `SPECS/etcd/generate_source_tarball.sh` for an example.